sentry-android-core/src/main/java/io/sentry/android/core/SessionTrackingIntegration.java

```diff
@@ -31,10 +31,21 @@ public void register(final @NotNull IHub hub, final @NotNull SentryOptions optio
             options.isEnableSessionTracking());
 
     if (options.isEnableSessionTracking()) {
-      watcher = new LifecycleWatcher(hub, options.getSessionTrackingIntervalMillis());
-      ProcessLifecycleOwner.get().getLifecycle().addObserver(watcher);
-
-      options.getLogger().log(SentryLevel.DEBUG, "SessionTrackingIntegration installed.");
+      try {
```

Comment:
This is a nice feature but is it worth the cost of using reflection?

Reply:
it's necessary if release health is enabled and androidx is not available, it's gonna crash.
this could be fixed by the clients on development, of course, but it requires code on their end and it makes the one-liner SDK integration a lie due to their restrictions.
ideally, we'd extract this as a separate maven package, as this would be needed for automatic breadcrumbs too, it's a temporary solution though.
they would work as the sentry ndk module, where reflection is necessary due to being sibling's modules.

Reply:
sounds good